src/config.js

```diff
@@ -42,6 +42,12 @@ export const _DEFAULT_TOKEN = {
   ...hathorLib.constants.DEFAULT_NATIVE_TOKEN_CONFIG,
 };
 
+/**
+ * Whether we should skip the words confirmation screen during
+ * new wallet creation
+ */
+export const SKIP_SEED_CONFIRMATION = false;
```

Comment:
This should be `true`, right?

Reply:
It should.

When I opened this PR was thinking on opening it to `master`, but I guess we can cherry-pick it in the future and set it to false again

Reply:
Fixed